curation-migration-datasync/database/dataService.ts

```diff
@@ -0,0 +1,130 @@
+import { Knex } from 'knex';
+import {
+  CuratedFeedItem,
+  CuratedFeedProspectItem,
+  CuratedFeedQueuedItems,
+  TileSource,
+} from '../types';
+import { queries } from './dbClient';
+import config from '../config';
+
+export class DataService {
+  private db: Knex;
+  constructor(db: Knex) {
+    this.db = db;
+  }
+
+  /**
+   * inserts into curated_feed_prospects table.
+   * unique index on (feed_id and resolved_id)
+   * @param trx
+   * @param prospectItem
+   */
```

Comment:
For all the functions in this file, is it possible to document the return type?

Reply:
done